storage/src/index/storage.rs

```diff
@@ -78,6 +78,8 @@ pub struct Cursor<'a, T: Translator, V: Eq> {
 
     entry: Option<OccupiedEntry<'a, T::Key, Record<V>>>,
     past: Option<Box<Record<V>>>,
+    past_tail: Option<*mut Record<V>>,
```

Comment:
Probably worth adding some comments as it's no longer clear what these three `past*` vars are for.

Reply:
maybe not lol

Reply:
added!